src/lib/linkedin.ts

```diff
@@ -458,7 +463,30 @@ export default class LinkedInAPI {
           attributes: mentionedAttributes,
           text: message.text || '',
         },
-        renderContentUnions: attachments,
+        renderContentUnions: [
+          ...attachments,
+          ...(options.quotedMessageID
+            ? (() => {
+              const cachedMessage = this.messagesCache.get(options.quotedMessageID)
```

Comment:
could we use texts.getOriginalObject like in platform-discord:

<img width="1189" alt="Screenshot 2024-05-23 at 9 13 06 PM" src="https://github.com/TextsHQ/platform-linkedin/assets/1093313/fd01fcc2-ede2-44f9-8689-d7f0040f5b66">